### DISCUSSION TITLE
Excuse me: Can livewrie use bootstrap-tagsinput?

### DISCUSSION THREAD

**Discussion by AngryLeaves** [livewire/livewire#2197] [1. Help]
I made a mistake and the problem has been solved by myself

**Comment by iamprincesly** [comment]
> I made a mistake and the problem has been solved by myself

How did you fix it. I have the same problem.

**Reply by AngryLeaves** [reply]
I gave up the idea of using tagsinput, and instead clicked on the li tag to pass the value to the hidden input through js for submission. If you want to use tagsinput,  https://forum.laravel-livewire.com/t/how-to-make-input-tags-in-livewire/2889   ，this should help you
